requirements.txt

```diff
@@ -6,3 +6,4 @@ python-dateutil
 pyyaml
 oauth2client
 ipaddress
+ansible
```

Comment:
I suspect this should probably be moved to test-requirements.txt, The module_utils libraries are actually shipped to the host system as part of the ansiballz package that contains the module code. For testing (and test.py currently) Ansible needs to be installed since we will want to avoid running the tests against a remote system through Ansible.